src/biokbase/narrative/staging/helper.py

```diff
@@ -0,0 +1,121 @@
+
+import os
+import urllib
+import urllib2
+import json
+
+"""
+KBase staging.helper class
+"""
+__author__ = "Tianhao Gu <tgu@anl.gov>"
+
+
+class Helper(object):
+
+    def __token(self):
```

Comment:
There's an auth client you can use that'll provide this for you - it's in biokbase.auth, and you can invoke with:

```import biokbase.auth
token = biokbase.auth.get_auth_token()
```

Reply:
This'll return None if there's not token, or the current token otherwise.

Reply:
👍